css-form.css

```diff
@@ -1,4 +1,4 @@
-.name{
+.base-class{
```

Comment:
why base-class? is it really a class? 🤔  what if we change the name by something like 'form' I was thinking a name but nothing comes to mind hehe

Reply:
and it's missign a space before {